### PR TITLE
Extra props

### PR DESCRIPTION
**Pull request by kevincasey1222**
Hi @mknoedel !

Here are lots of extra properties per request from NextRoll, INT-1652.

--Kevin

### CODE REVIEW

File: src/sync/converters.ts [hunk 1]
```diff
@@ -349,6 +399,35 @@ export function createRepoAllowsUserRelationship(
...
+    _mapping: {
+      sourceEntityKey: prKey,
+      relationshipDirection: RelationshipDirection.REVERSE,
+      targetFilterKeys: [['_type', 'login']],
+      targetEntity: {
+        _class: 'User',
+        _type: GITHUB_MEMBER_ENTITY_TYPE,
+        login: unknownLogin,
+      },
+      skipTargetCreation: true,
```
Comment (mknoedel) [inline src/sync/converters.ts:425]: This needs to be false for the comment to be true.

File: src/sync/converters.ts [hunk 2]
```diff
@@ -376,6 +455,19 @@ export function toPullRequestEntity(
...
   );
+  const commitsCount = commits ? commits.length : 0;
+  const approvalsCount = reviews ? reviews.filter(isApprovalReview).length : 0;
+
+  let approvalLastAt: number | undefined = undefined;
+  if (approvedCommits) {
+    const commitTimes = approvedCommits?.map(
+      (c) => parseTimePropertyValue(c.authoredDate) || 0,
+    );
+    const maxTime = Math.max(...commitTimes);
```
Comment (mknoedel) [inline src/sync/converters.ts:466]: Does this throw an error with a spread empty array?

**Review by mknoedel — COMMENTED** [review]
Looks great! A few small comments

File: src/sync/converters.ts (hunk 1 above)
Comment (kevincasey1222) [reply src/sync/converters.ts:425]: Good catch! Thanks!

File: src/sync/converters.ts (hunk 2 above)
Comment (kevincasey1222) [reply src/sync/converters.ts:466]: Also good catch! Thanks!

**Review by mknoedel — APPROVED** [review]

**Pull request merged**